.gitignore

```diff
@@ -0,0 +1,18 @@
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
```

Comment:
i think these files are related with c languages so its better to use global gitignore  so only go related git ignore will be placed :) Please make necessary adjustments @Aju100

Reply:
sure

Reply:
@ujwaldhakal  can u review code.

Reply:
@Aju100 can you rebase with master and squash 3 commits into 1 so i can merge it

Reply:
ujwaldhakal i am not able to rebase and squash 3 commits into 1 . But everything is right, isn't it?

Reply:
yes just it makes more commit and hard for review ok let me do it for you

Reply:
git pull --rebase origin master & git rebase -i HEAD~${numberofcommits}

Reply:
git pull Aju100 master && git rebase -i HEAD~4

Reply:
ok ujwaldhakal I will do